experimental/GaloisGrp/Solve.jl

```diff
@@ -316,16 +366,60 @@ function as_radical_extension(K::NumField, aut::Map, zeta::NumFieldElem)
   end
   s = coeff(r^d, 0)
   @hassert :SolveRadical 1 s == r^d
-  L, b = number_field(gen(parent(defining_polynomial(K)))^d-s, cached = false, check = false)
+  L, b = number_field(gen(parent(defining_polynomial(K)))^d-s, cached = false, check = CHECK)
   @assert base_field(L) == base_field(K)
-  return L, hom(L, K, r)
+  global last_b = (L, K, r)
+  return L, hom(L, K, r, check = CHECK)
 end
 
 function Oscar.solve(f::fmpq_poly; max_prec::Int=typemax(Int))
   return solve(numerator(f), max_prec = max_prec)
 end
 
+@doc Markdown.doc"""
+    Oscar.solve(f::fmpz_poly; max_prec::Int=typemax(Int))
+    Oscar.solve(f::fmpq_poly; max_prec::Int=typemax(Int))
```

Comment:
Isn't this exported?
```suggestion
    solve(f::fmpz_poly; max_prec::Int=typemax(Int))
    solve(f::fmpq_poly; max_prec::Int=typemax(Int))
```

Reply:
no can do: solve is not imported...